src/NosCore.GameObject/Networking/LoginService/LoginService.cs

```diff
@@ -84,6 +85,11 @@ public class LoginService : ILoginService
                     return;
                 }
 
+                if(useApiAuth)
+                {
+                    username = await _authHttpClient.GetAwaitingConnection(null, passwordToken, clientSession.SessionId);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Introduce a new variable instead of reusing the parameter 'username'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262030)